ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -1064,3 +1083,28 @@ Check Query Response Values    # robocop:disable
     IF    ${checked} == ${FALSE}
         Fail    msg=The metric you are looking for has not been found. Check the query parameter and try again 
     END
+
+Traffic Should Be Redirected Based On Canary Percentage
+    [Documentation]    Sends an arbitrary number of queries ${total} and checks the amount of
+    ...                them which gets redirected to the given ${model_name}
+    ...                matches the expected probability ${exp_percentage}.
+    ...                It applies an arbitrary toleration margin of ${toleration}
+    [Arguments]    ${exp_percentage}    ${isvc_name}    ${model_name}    ${namespace}
+    ${total}=    Set Variable    ${20}
+    ${hits}=    Set Variable    ${0}
+    ${toleration}=    Set Variable    ${20}
+    FOR    ${counter}    IN RANGE    ${0}    ${total}
+        Log    ${counter}
+        ${status}=    Run Keyword And Return Status
+        ...    Query Model Multiple Times    isvc_name=${isvc_name}    model_name=${model_name}    n_times=1
+        ...    namespace=${namespace}
+        IF    ${status} == ${TRUE}
```

Comment:
Do you think it makes sense to follow robocop advise to simplify this?
```suggestion
        IF    ${status}
```

Reply:
nope, I want the variable to accept boolean values only. 
`IF ${status}` is verified when the variable is not None

Reply:
Just to be sure - I suppose that by `verified` you mean that the expression is resolved as true, right?

In this case I think I'm missing something. Based on the documentation from the `Run Keyword And Return Status` keyword, I understand that the status is always boolean only. So is this just a precaution in case of bad code change in the future?

Reply:
>Just to be sure - I suppose that by verified you mean that the expression is resolved as true, right?

yes


> I understand that the status is always boolean only


Ah, sorry I mixed it up with previous comment which is the same but on a different line where the variable is taken as argument. In this particular case we could remove the `== ${TRUE}`. In general i think it's one of my coding habits to explicitly check for the value even if it's True.

Reply:
Okay, I don't insist. Feel free to let it the way it is, I guess. Just wanted to check whether I'm not missing anything :upside_down_face: :slightly_smiling_face: